aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -377,37 +377,58 @@ builder.On<string, string>("ReceiveMessage", (user, message) => ...
 await builder.StartAsync();
 ```
 
-### SignalR seamless reconnect
+### SignalR stateful reconnect
 
-SignalR seamless reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
 
-Seamless reconnect achieves this by:
+Stateful reconnect achieves this by:
 
 * Temporarily buffering data on the server and client.
 * Acknowledging messages received (ACK-ing) by both the server and client.
 * Recognizing when a connection is returning and replaying messages that may have been sent while the connection was down.
 
-To opt-in to seamless reconnect:
+To opt-in to stateful reconnect for a JavaScript or Typescript client:
 
-* Update the .NET client code to enable the `UseAcks` option:
+* Update the JavaScript or TypeScript client code to enable the `withStatefulReconnect` option:
+
+  ```JavaScript
+  const builder = new signalR.HubConnectionBuilder()
+    .withUrl("/hubname")
+    .withStatefulReconnect({ bufferSize: 1000 });  // Optional, defaults to 100,000
+  const connection = builder.build();
+  ```
+
+* Update the server hub endpoint configuration to enable the `AllowStatefulReconnects` option:
+
+  ```csharp
+  app.MapHub<MyHub>("/hubName", options =>
+  {
+      options.AllowStatefulReconnects = true;
+  });
+  ```
+
+To opt-in to stateful reconnect for a .NET client:
+
+* Update the .NET client code to enable the `UseStatefulReconnect` option:
 
   ```csharp
-  var hubConnection = new HubConnectionBuilder()
-      .WithUrl("<hub url>",
-               options =>
-               {
-                  options.UseAcks = true;
-               })
-      .Build();
+    var builder = new HubConnectionBuilder()
+        .WithUrl("<hub url>")
+        .WithStatefulReconnect();
+    builder.Services.Configure<HubConnectionOptions>(o => o.StatefulReconnectBufferSize = 1000);
+    var hubConnection = builder.Build();
   ```
 
-* Update the server hub endpoint configuration to enable the `AllowAcks` option:
+* Update the server hub endpoint configuration to enable the `AllowStatefulReconnects` option:
 
   ```csharp
-  app.MapHub<AppHub>("/default", o => o.AllowAcks = true);
+  app.MapHub<MyHub>("/hubName", options =>
+  {
+      options.AllowStatefulReconnects = true;
+  });
   ```
 
-For more information on the progress of the seamless reconnect feature for ASP.NET Core 8.0, see [dotnet/aspnetcore #46691](https://github.com/dotnet/aspnetcore/issues/46691).
+For more information on the progress of the SignalR stateful reconnect feature for ASP.NET Core 8.0, see [dotnet/aspnetcore #46691](https://github.com/dotnet/aspnetcore/issues/46691).
```

Comment:
This should be removed now that 8.0 is done?

Reply:
I had purposely left it in until I had a doc update to point to instead more details, which I am working on next.  
I went ahead and removed it now.  I will add a link back in once I have the doc update in to point to.
Thanks.